set.mm

```diff
@@ -264077,198 +264256,394 @@ the same dimension over the same (nonzero) ring.  (Contributed by AV,
+        UYIMURUSZUYJVTURZUYKVTURZUYMVTURZUYNUYPVOZUYHMVTUYIKUYHUYCMVTKVAUXNUYCU
+        XJAUYCUXIRVPVPEGKMPVQVKZVRZUYHMVTUYIUWBUYHUWBEURMVTUWBVAUYHFEUWBUYBAUXI
+        UXJVSVIEGUWBMPVQVKZVRZUYHMVTUYIUVPUYHUVPEURMVTUVPVAUYHUWKEUVPUWJBEWAUXN
+        UXJWBZVIEGUVPMPVQVKZVRZUYRUYJWCURUYSUYKWCURUYTUYMWCURVUAUYJWDUYKWDUYMWD
+        UYJUYKUYMWEWFWGWHUYHUDMUYLUYMUFUWIUVPWIWIVTUYHMUVPUWKVUFUYHMVTUVPVUGWJW
+        LZUYQUYJUYKUFWKVUHUYHUDMUYJUYKUFKUWBWIVTVTVUIVUCVUEUYHUDMVTKVUBWMZUYHUD
+        MVTUWBVUDWMZWNUYHUDMVTUVPVUGWMZWNUYHUDMUYOUYKUFUVRUWBWIWIVTVUIUYQUYJUYM
+        UFWKVUEUYHUDMUYJUYMUFKUVPWIVTVTVUIVUCVUHVUJVULWNVUKWNWOZUXNUXTUXJUXOUWK
+        URUYDBEUWKGUWIMUVPPUYEVEVFWPUXNUCUWKUXOUWCVUMWHUXCJUWKUYGUMVOUXNIJUWKNU
+        YGJNWQIUWPNWRIUWPNXBWSWTIUWPUWCNUUAXAZXCXDUUBUUDZUUEZUUCAFDUWAFFUUFZUCH
+        LXEXEUWDLXFVNZSVURVJZTAFUYACEUKZXEQAUYCVUTXEURRCEGKMPXGVKXHZAUXFUSZUVRE
+        URUWAXEURZVVBFEUVRUYBAUYCUXFUVRFURRCEFGKMUVPPQVEVFZVIBEGUVRMPXGVKZVUPAF
+        XEURZVVFVUQXEURVVAVVAFFUUGUUHZAUXHUVPUWBVUQUJZXIUWDVURVOZAUXHUSZVVHVVIV
+        VJUXFUXIVVHAUXFUXGXJVVJUXIUXJUXMXMUVPUWBFFXKXLXNXOXPAFDUWKVUQHUCLXEXEUW
+        DVURSVUSTVVAUXNUXTUWKXEURZUYDBEGUWIMPXGVKZVUOVVGAUXKUWBUVPVUQUJZXIVVIAU
+        XKUSZVVMVVIVVNUXIUXFVVMAUXIUXJXJVVNUXFUXGABCEFGKMUWBUVPPQRUPXMUWBUVPFFX
+        KXLXNXOXPYEAUXBLUXAUCFUVRUMZUTZUNUQUWHAFDFUXALVVOXEVURSVUSTVVAAFDUWGVVO
+        XQZUTZVAFDUXAVAAFFDUWGVVQAUCFUWFDVVBUWADUWELXEVURSVUSALUUIURZUXFTVPVVEV
+        VBHUWAUWDDAUXFUXGUXLVUPVBVCVVBUWEWIURZUWEXRZVURWIURZVVCUWEVURXSUQZUWAXT
+        ZUWEVURYAUJVVBEWIURZUWAWIURVVTVWEVVBGUEXQUUPUUJXEURGVTMYBUQEPVTMYBYCUUK
+        ZWTUVTBEWIYDHUWAUWDWIYFYNVWAVVBHUWAUWDYGWTVVBLXFYHVVEVWDVVBVWCUWEYIUWAU
+        WEVURYJHUWAUWDUWEUWEVJYKYOWTUWAUWEWIWIVURYLYMUULVCAFFVVOVHZFFVVQVHFFVVQ
+        VAAUYCVWGRUCCEFGKMPQVGVKZFFVVOUUMFFVVQVLYNVMAFDVVRUXAAVVPVVQUTZUXAUUNFY
+        PZUTZVVRUXAAVWIUXAVVOVVQUTZUTVWKUXAVVOVVQUUOAVWLVWJUXAAVWGVWLVWJVOVWHFF
+        VVOUUQVKUURUVFAVVPUWGVVQAUCJFFUVRUWTUWFVVOUXAVVDAVVOYQAUXAYQUWPUVRVOZUW
+        SUWELUNVWMHUWROUWAUWDVWMUWQUVTBEUWPUVRUVOUVSUUSUUTVWMOIUWPUWBUVQUQZNULU
+        WDIVWNNOUWPUWBUVQYCUBUVAVWMIVWNUWCNUWPUVRUWBUVQUVBUVCUVDUVEYRXAZUVGVWKU
+        XAVOAVWKUXAFYPZUXAUXAFUVHFFXTVWPUXAVOFUVIJFFUWTUVJUVKUVLWTYEXCXDAUXAWIU
+        RUXAXRZVWBVVFUXAVURXSUQZFXTZUXAVURYAUJAJFUWTWIAFVUTWIQVWEVUTWIURAVWFUYA
+        CEWIYDUVMXHUVNVWQAJFUWTYGWTALXFYHVVAVWSAVWRUXAYIFUXAVURYJJFUWTUXAUXAVJY
+        KYOWTFUXAWIWIVURYLYMVWHYSAVVPUWGLUNVWOYRYTAUXEUWNLUNAHFUXDUWMUXNUXDLUXQ
+        UNUQUWMUXNUWKDUWKUXCLUXPXEVURSVUSAVVSUXITVPVVLUXRUXNUXCWIURZUXCXRZVWBVV
+        KUXCVURXSUQZUWKXTZUXCVURYAUJUXNVWEUWKWIURVWTVWEUXNVWFWTUWJBEWIYDIUWKNWI
+        YFYNVXAUXNIUWKNYGWTUXNLXFYHVVLVXCUXNVXBUXCYIUWKUXCVURYJIUWKNUXCUXCVJYKY
+        OWTUWKUXCWIWIVURYLYMUYFYSUXNUXQUWLLUNVUNYRYTWHYRWO $.
+    $}
+
+    $( Once these OLD theorems are deleted, the above curly braces become
+       unnecessary, and can be deleted. $)
```

Comment:
Cannot the curly braces be adjusted now (possibly by having shorter scopes, which is often a better practice anyway since long scopes are error-prone), so that we do not forget it next year when we remove the OLD versions ?